pkg/scene/caption.go

```diff
@@ -0,0 +1,127 @@
+package scene
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/text/language"
+
+	"github.com/asticode/go-astisub"
+)
+
+var CaptionExts = []string{"vtt", "srt"} // in a case where vtt and srt files are both provided prioritize vtt file due to native support
```

Comment:
Any supported subtitle type will be parsed upon load and converted to vtt. Even vtt's will be parsed (and dropped if they are not valid) and converted. So they are not technically `natively` supported. I think it is more optimal to prefer subtitles based on format popularity. That will mean that they are stated and returned first during playback (so 1 stat will probably be enough). Idk what's the most used format though for our specific `content`. For other content its usually srt -> ass -> ... . For anime content ass -> srt -> ....  ( I think we should add `ass` as well btw, maybe after testing it a bit ?)

Reply:
From what I understand there are nuances across different caption file types that astisub will try to convert into the VTT standard. I'm not sure the astisub library can guarantee perfect conversions. A few weeks ago, a post I read suggested that some conversion methods aren't perfect. I know we are calling `WriteToWebVTT()` for VTT files as well, but I would think the library wouldn't change much, if anything, from the original VTT file. 

Regarding common formats, Adulttime is the only studio I am aware of which provides captions. The format they use is VTT. There could be others I'm not aware of but I think most studios would be using videojs as their video player so they would likely be providing the VTT format.